applications/nfc/helpers/nfc_generators.c

```diff
@@ -333,6 +460,26 @@ static const NfcGenerator ntag_i2c_plus_2k_generator = {
     .generator_func = nfc_generate_ntag_i2c_plus_2k,
     .next_scene = NfcSceneMfUltralightMenu};
 
+static const NfcGenerator mifare_classic_1k_4b_uid_generator = {
+    .name = "Mifare Classic 1k 4byte UID",
+    .generator_func = nfc_generate_mf_classic_1k_4b_uid,
+    .next_scene = NfcSceneMfClassicMenu};
+
+static const NfcGenerator mifare_classic_1k_7b_uid_generator = {
+    .name = "Mifare Classic 1k 7byte UID",
+    .generator_func = nfc_generate_mf_classic_1k_7b_uid,
+    .next_scene = NfcSceneMfClassicMenu};
+
+static const NfcGenerator mifare_classic_4k_4b_uid_generator = {
+    .name = "Mifare Classic 4k 4byte UID",
+    .generator_func = nfc_generate_mf_classic_4k_4b_uid,
+    .next_scene = NfcSceneMfClassicMenu};
+
+static const NfcGenerator mifare_classic_4k_7b_uid_generator = {
+    .name = "Mifare Classic 4k 7byte UID",
+    .generator_func = nfc_generate_mf_classic_4k_7b_uid,
+    .next_scene = NfcSceneMfClassicMenu};
```

Comment:
Please add a comma to the last line of each generator to make it more readable.

Reply:
Fixed